tests/test_benchmark.py

```diff
@@ -183,7 +183,7 @@ def run(self, args):
 
   def get_output_files(self):
     ret = [self.filename]
-    if 'WASM=1' in self.extra_args:
+    if 'WASM=0' not in self.extra_args:
```

Comment:
actually this might read easier if we flip the conditional here:
```python
if 'WASM=0' in self.extra_args:
  ret.append(self.filename + '.mem')
else:
  ret.append(self.filename[:-3] + '.wasm')
```
which is nicer than the double-negative "if not no wasm"

Reply:
Good idea, done.